web/src/user_events.js

```diff
@@ -65,6 +65,7 @@ export const update_person = function update(person) {
         pm_list.update_private_messages();
         if (people.is_my_user_id(person.user_id)) {
             current_user.full_name = person.full_name;
+            current_user.name_with_diacritics_removed = undefined;
```

Comment:
It already happens in `people.set_full_name` above, so I guess I just can remove this line then.

Reply:
I think if I remove this, `current_user` won't have 'name_with_diacritics_removed' reset, but it also seems like `current_user` isn't ever being passed to `build_termlet_matcher` (it would be helpful to get a second pair of eyes to confirm that) so it should be fine.

Reply:
Yeah, only items in `people.ts` would be passed there; `current_user` is a special data structure is not directly related to user objects.